torchaudio/models/wav2vec2/components.py

```diff
@@ -273,15 +275,22 @@ def forward(
         self,
         x: Tensor,
         attention_mask: Optional[Tensor] = None,
-    ) -> Tensor:
+        position_bias: Optional[Tensor] = None,
+        key_padding_mask: Optional[Tensor] = None,
+    ) -> Tuple[Tensor, Optional[Tensor]]:
         """
         Args:
             x (Tensor): shape: ``[batch_size, sequence_length, embed_dim]``.
-            attention_mask (Tensor or None, optional):
+            attention_mask (Tensor or ``None``, optional):
                 shape: ``[batch_size, 1, sequence_length, sequence_length]``
-
+            position_bias (Tensor or ``None``): Has to be ``None``, necessary for compatibility
+                with :py:class:`WavLMSelfAttention`. (Default: ``None``)
```

Comment:
Since it is not used, there is no use of adding annotation in docstring.

```suggestion
            position_bias: Not used. Only for the compatibility with :py:class:`WavLMSelfAttention`.
```